src/pages/lobby.tsx

```diff
@@ -31,17 +34,27 @@ const Lobby: FC = () => {
 
   api.lobby.onReadyToPlay.useSubscription(undefined, {
     onData({ roomId }) {
-      void push({ pathname: pages.match, query: { roomId } });
+      setTrack("MatchMakingOutro");
```

Comment:
Why not just use a handler that just plays the sound instead of setting a variable that then changes a value passed to a hook that plays the sound?

Reply:
Same way `playSfx` works